lib/screens/siteConfig/CertificateScreen.dart

```diff
@@ -219,34 +219,22 @@ class _CertificateScreenState extends State<CertificateScreen> {
           ConfigButtonItem(
               content: Center(child: Text('Choose a file')),
               onPressed: () async {
-                var file;
                 try {
-                  await FilePicker.clearTemporaryFiles();
-                  file = await FilePicker.getFile();
-
-                  if (file == null) {
-                    print('GOT A NULL');
+                  final content = await Utils.pickFile(context);
+                  if (content == null) {
                     return;
                   }
-                } catch (err) {
-                  print('HEY $err');
-                }
 
-                var content = "";
-                try {
-                  content = file.readAsStringSync();
+                  _addCertEntry(content, (err) {
```

Comment:
It seems potentially unnecessary to move this into the try`` - errors should invoke the callback here? I don't see any harm though.